app/utils/filter.scala

```diff
@@ -71,5 +77,8 @@ object ResourceFilter {
     ResourceFilter((defaultKeys ++ filterKeys).toMap)
   }
 
-  lazy val all: Matchable[JsValue] = (_: JsValue) => true
+  lazy val all: Matchable[JsValue] = new Matchable[JsValue] {
+    override def isMatch(value: JsValue): Boolean = true
+    override def inverse: Boolean = false
+  }
```

Comment:
What are these changes for?

Reply:
ah - I knew there was something I still needed to test. I'll document it and ping you.

Reply:
I've now extracted the filtering changes into their own PR #214 for clarity and ability to independently roll either back.